sw/device/lib/testing/test_status.c

```diff
@@ -0,0 +1,35 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/test_status.h"
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/log.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/runtime/hart.h"
+
+/**
+ * Address of the memory location to write the test status. For DV use only.
+ */
+static const uintptr_t kSwDvTestStatusAddr = 0x1000fff8;
+
+inline void set_test_status(const test_status_t test_status) {
+  if (kDeviceType != kDeviceSimDV)
```

Comment:
I am aware that files under `sw/device/lib/arch/` currently contain only device-specific constants but I wonder if we could move these implementations to `device_sim_dv.c` and have only **weak** NOP implementations here. @mcy and @gkelly, does that make sense?

Reply:
I have changed the implementation - PTAL.